### ISSUE TITLE
LSP completion repeats character in gdscript

### ISSUE THREAD

**Issue by rcorre** [helix-editor/helix#5878]
### Summary

I'm editing the following `.gd` file:

```
extends Node

func _ready() -> void:
	var n := Node.new()
```

Below the last line, I type `if n.h<tab><tab>` quickly. It completes to `if n.hhas_method` instead of `if n.has_method`. This does not happen if I press `<tab>` more slowly.

### Reproduction Steps

 

I tried this:

1. `hx Example.gd` 
2. `ge`
3. `O`
4. `<tab>`
5. `if n.h<tab><tab>` (quickly)

I expected this to happen:
```if n.has_method```
Instead, this happened:
```if n.hhas_method```

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>
[helix.log](https://github.com/helix-editor/helix/files/10680425/helix.log)

For comparison, here's the nvim LSP log for the same operation:

[nvim.log](https://github.com/helix-editor/helix/files/10680429/nvim.log)

### Platform

Linux

### Terminal Emulator

st 0.8.5

### Helix Version

22.12-231-g23ed8c12

**Comment by sudormrfbin** [comment]
Fixed by #1819, and I think there was already an issue open about it but can't find it right now.

**Comment by sudormrfbin** [comment]
Duplicate of #4851.
